src/communication/bolt/client.cpp

```diff
@@ -0,0 +1,247 @@
+// Copyright 2022 Memgraph Ltd.
```

Comment:
I mostly copied this file over, but after that changed a few things, check the commit history.